Its.Log.Monitoring/TestDefinition.cs

```diff
@@ -32,7 +38,15 @@ internal static TestDefinition Create(MethodInfo methodInfo)
                 new object[] { methodInfo },
                 null);
             testDefinition.TestType = testType;
+            testDefinition.TestParameters = methodInfo.GetParameters().Select(p => 
+            new TestParameter(p.Name, p.DefaultValue));
             return testDefinition;
         }
+
+        public IEnumerable<TestParameter> TestParameters
+        {
+            get { return testParameters ?? System.Linq.Enumerable.Empty<TestParameter>(); }

```

Comment:
It's better to assign and return or you may not get the same instance each time, which can lead to some unexpected behaviors.